presto-hive/src/test/java/com/facebook/presto/hive/HiveTestUtils.java

```diff
@@ -175,6 +177,14 @@ public static Set<HiveRecordCursorProvider> getDefaultHiveRecordCursorProvider(H
                 .build();
     }
 
+    public static Set<HiveRecordCursorProvider> getDefaultS3HiveRecordCursorProvider(HiveClientConfig hiveClientConfig, MetastoreClientConfig metastoreClientConfig)
+    {
+        HdfsEnvironment testHdfsEnvironment = createTestHdfsEnvironment(hiveClientConfig, metastoreClientConfig);
+        return ImmutableSet.<HiveRecordCursorProvider>builder()
```

Comment:
Minor: this could just be `ImmutableSet.of(...)`.

Reply:
I agree it could be replaced, but this is maintaining the pattern in [this class](https://github.com/prestodb/presto/blob/2910a8818ffbd49cdb5ba411751c4c340ebe7c7d/presto-hive/src/test/java/com/facebook/presto/hive/HiveTestUtils.java#L147-L186).

Reply:
Makes sense, feel free to leave it as-is.